JesusFilm/core
# fix: save inline edit block on outside click

# Description

Open for anyone to review - this is a cooldown 5 task I picked up while waiting for other work to be reviewed

[Save text content changes on when clicking outside the block](https://3.basecamp.com/3105655/buckets/28141725/todos/5055615357)

# How should this PR be QA Tested?

- [ ] Manually test below on Typography, Button, RadioOption & SignUp blocks on mobile and desktop
> - [ ] Edit text, click **outside the card**, text should save
> - [ ] Edit text, click **outside the element** (but still inside the card), text should save
> - [ ] Edit text on card **without a background**, click Done button, text should save
> - [ ] Edit text on card **with a background image**, click Done button, text should save
> - [ ] Edit text on card **with a background video**, click Done button, text should save

# Checklist
